code/__DEFINES/traits/declarations.dm

```diff
@@ -602,6 +602,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_IMMERSE_STOPPED "immerse_stopped"
 /// The effects of hyperspace drift are blocked when the tile has this trait
 #define TRAIT_HYPERSPACE_STOPPED "hyperspace_stopped"
+///Chasms will be safe to cross if there is something with this trait on it
+#define TRAIT_CHASM_STOPPER "chasm_stopper"
```

Comment:
Put this one two defines up right by `TRAIT_CHASM_STOPPED`

Reply:
Also, with the existence of [`TRAIT_CHASM_STOPPED`], you may not even need [`TRAIT_CHASM_STOPPER`]? 

You can just give atoms `TRAIT_CHASM_STOPPED` when they enter a tram and remove it when they exit, I think you can do it in `add_item_on_transport` / `remove_item_from_transport`

Reply:
chasm_stopped is for turfs

Reply:
Can't you use the `give_turf_traits` element instead of adding a new redundant trait?

Reply:
Trust me I wouldnt do it if it worked

Reply:
I see. Tram movement is a bit niche.